docs/manual/lcos.qbk

```diff
@@ -20,11 +20,73 @@ special different ways adapted to HPX.
 
 # barrier
 
-# and_gate
+[heading Channels]
 
-# barrier
+Channels combine communication (the exchange of a value) with
+synchronization (guaranteeing that two calculations (tasks) are in a known
+state). A channel can transport any number of values of a given type from
+a sender to a receiver:
+
+    hpx::lcos::local::channel<int> c;
+    c.set(42);
+    cout << c.get();      // will print '42'
+
+Channels can be handed to another thread (or in case of channel components,
+to other localities), thus establishing a communication channel between two
+independed places in the program:
+
+    void do_something(
+        hpx::lcos::local::receive_channel<int> c,
+        hpx::lcos::local::send_channel<> done)
+    {
+        cout << c.get();        // prints 42
+        done.set();             // signal back
+    }
+
+    {
+        hpx::lcos::local::channel<int> c;
+        hpx::lcos::local::channel<> done;
```

Comment:
They could match. `receive_channel` and `send_channel` are constructable from `channel`, but they limit its use to one end of the communication. Thus this would work as well:

    void do_something(  
        hpx::lcos::local::channel<int> c,  
        hpx::lcos::local::channel<> done)  
    { ... }